arangod/VocBase/LogicalCollection.cpp

```diff
@@ -1051,7 +1051,7 @@ bool LogicalCollection::dropIndex(IndexId iid) {
 
   arangodb::aql::QueryCache::instance()->invalidate(&vocbase(), guid());
 
-  bool result = _physical->dropIndex(iid);
+  bool result = _physical->dropIndexFast(iid).ok();
```

Comment:
I would prefer if we changed this place and make `result` a `Result` here, no boolean.

Reply:
I can change that, but I will not modify the caller code except for adding a `.ok()`, ok?